src/GameData.ts

```diff
@@ -0,0 +1,320 @@
+import fetch from "node-fetch";
+import Discord = require("discord.js");
+import { levenshteinDistance, levenshteinDistanceArray } from "./LevenshteinDistance";
+import { SharedSettings } from "./SharedSettings";
+
+interface ChampionData {
+    id: number;
+    name: string;
+    key: string;
+    skins: SkinData[];
+}
+
+interface SkinData {
+    id: number;
+    name: string;
+    splashPath: string;
+    uncenteredSplashPath: string;
+    tilePath: string;
+    loadscreenPath: string;
+    chromas: ChromaData[];
+}
+
+interface ChromaData {
+    id: number;
+    name: string;
+    chromaPath: string;
+    colors: string[];
+}
+
+interface PerkData {
+    id: number;
+    name: string;
+    shortDesc: string;
+    iconPath: string;
+    endOfGameStatDescs: string[];
+}
+
+interface ItemData {
+    id: number;
+    name: string;
+    categories: string[];
+    price: number;
+    priceTotal: number;
+    iconPath: number;
+    from: string[];
+    to: string[];
+}
+interface SearchType {
```

Comment:
This is a bad name. SearchType makes me think it is an enum of different sorting strategies, not a container that has items and a score.